velero-plugin-for-aws/config.go

```diff
@@ -4,14 +4,15 @@ import (
 	"context"
 	"crypto/tls"
 	"crypto/x509"
+	"net/http"
+	"os"
+
 	"github.com/aws/aws-sdk-go-v2/aws"
```

Comment:
These changes aren't related or needed to fix the commit, so we should leave this as it is from upstream -- just remove the file from the PR.